src/Builtins.jl

```diff
@@ -170,19 +171,52 @@ function show(io::IO, ::MIME"text/html", select::Select)
     withtag(io, :select) do
         for o in select.options
             print(io, """<option value="$(htmlesc(o.first))"$(select.default === o.first ? " selected" : "")>""")
-            withtag(io, :option, :value=>o.first) do
-                if showable(MIME"text/html"(), o.second)
-                    show(io, MIME"text/html"(), o.second)
-                else
-                    print(io, o.second)
-                end
+            if showable(MIME"text/html"(), o.second)
+                show(io, MIME"text/html"(), o.second)
+            else
+                print(io, o.second)
+            end
+            print(io, "</option>")
+        end
+    end
+end
+
+get(select::AbstractSelect) = ismissing(select.default) ? first(select.options).first : select.default
+
+
+"""A multi-selector (`<select multi>`) - the user can choose one or more of the `options`, an array of `Strings.
+
+`options` can also be an array of pairs `key::String => value::Any`. The `key` is returned via `@bind`; the `value` is shown.
+
+See the [Mozilla docs about `select`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select)
+
+# Examples
+`@bind veg MultiSelect(["potato", "carrot"])`
+
+`@bind veg MultiSelect(["potato" => "🥔", "carrot" => "🥕"])`
+
+`@bind veg MultiSelect(["potato" => "🥔", "carrot" => "🥕"], default=["carrot"])`"""
+struct MultiSelect <: AbstractSelect
```

Comment:
Did you name it after the [REPL menu](https://docs.julialang.org/en/v1/stdlib/REPL/#MultiSelectMenu-1)? Nice!